lcls-twincat-motion/Library/POUs/Motion/Utils/F_MotionErrorCodeLookup.TcPOU

```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <POU Name="F_MotionErrorCodeLookup" Id="{71c436f2-8bc4-46b7-82a0-e1e338c02dd9}" SpecialFunc="None">
+    <Declaration><![CDATA[FUNCTION F_MotionErrorCodeLookup : STRING
+VAR_INPUT
+    nErrorId: UDINT;
+END_VAR
+VAR
+    msg: STRING;
+END_VAR
+]]></Declaration>
+    <Implementation>
+      <ST><![CDATA[CASE nErrorId OF
+    // Common NC errors
```

Comment:
👍

Reply:
Yeah, this is so nice...

Reply:
It's also something that should be in the built-in twincat MC libs